packages/plugins/script/write-constants.ts

```diff
@@ -93,8 +93,13 @@ const writeConstants = async () => {
     },
   }
 
-  const contractInfo = CHUGSPLASH_CONTRACT_INFO.map(
-    ({ artifact, constructorArgs, expectedAddress }) => {
+  const contractInfo = getChugSplashConstants(31337)
+    .filter(
+      (el) =>
+        el.artifact.contractName !== 'ChugSplashFunder' &&
+        el.artifact.contractName !== 'ChugSplashLZReceiver'
+    )
+    .map(({ artifact, constructorArgs, expectedAddress }) => {
```

Comment:
We don't need the funding-related contracts in the constants file since the Foundry plugin doesn't need to interact with them. So we use a default chain id when calling `getChugSplashConstants` and then filter out the funding-related contracts below so they don't even make it into the constants file.